packages/genesys-spark-components/src/components/beta/gux-phone-input/gux-phone-input.tsx

```diff
@@ -239,6 +248,10 @@ export class GuxPhoneInput {
   componentDidLoad(): void {
     this.setInput();
     this.setListBox();
+
+    if (this.hasError) {
+      this.inputElement.setAttribute('aria-invalid', 'true');
```

Comment:
Are we sure we will not be causing issues here when we are changing attributes on the slotted inputs?

I am not sure what alternative we have to updating the attribute but maybe we will need to have some documentation letting devs know we will be changing this attribute etc....